docs/source/concept_models/vivarium_us_cvd/concept_model.rst

```diff
@@ -341,9 +343,226 @@ Intended to identify groups that we are interested in being able to track and co
+
+
+**Screening Visit**
+
+.. image:: decision_tree_screening.svg
+
+.. list-table:: Screening Inputs
+  :widths: 3 15 15
+  :header-rows: 1
+
+  * - ID
+    - Decision Information 
+    - Notes
+  * - A
+    - Dependent on scenario, either 50% or 100%  
+    - For 50% scenario, assignment is random 
+  * - B
+    - LDL-c is tested if ASCVD score >= 5% 
+    - ASCVD = -19.5 + (0.043 * SBP) + (0.266 * Age) + (2.32 * Sex)
+  * - C
+    - If age>40 and LDL-c>70mg/dL, 85% receive treatment prescription 
+    - Type of treatment assignment below. 85% is random chance. 
+  * - D 
+    - If simulant is eligible, either 50% or 100% depending on scenario  
+    - For 50% scenario, assignment is random 
+
+
+**Follow-up Visit**
+
+.. image:: decision_tree_followup.svg
+
+.. list-table:: Followup Inputs
+  :widths: 3 15 15
+  :header-rows: 1
+
+  * - ID
+    - Decision Information 
+    - Notes
+  * - A
+    - Dependent on scenario, either 50% or 100%  
+    - For 50% scenario, assignment is random 
+  * - B
+    - If LDL-c>70mg/dL and patient is adherent 85% chance of increasing intensity 
+    - Further details below. 85% is random chance.
+  * - C
+    - If simulant is eligible, either 50% or 100% depending on scenario  
+    - For 50% scenario, assignment is random 
+
+
+**Emergency Visit**
+
+.. image:: decision_tree_emergency.svg
+
+
+**Blood Pressure Ramp - Initial Diagnosis**
+
+.. image:: sbp_ramp_initial.svg
+
+.. list-table:: Blood Pressure Ramp Initial Diagnosis 
+  :widths: 3 15 15
+  :header-rows: 1
+
+  * - ID
+    - Decision Information 
+    - Notes
+  * - A
+    - NEEDED  
+    -  
+  * - B
+    - If simulant is eligible, either 50% or 100% depending on scenario  
+    - For 50% scenario, assignment is random 
+  * - C
+    - NEEDED  
+    -  
+
+
+.. todo::
+  - Unclear how patients are assigned mono or combo therapy and probability of assignment 
+  - Need to figure out how treatment effect works 
+
+
+**Blood Pressure Ramp - Follow-up**
+
+.. image:: sbp_ramp_followup.svg
+
+.. list-table:: Blood Pressure Ramp Follow-up 
+  :widths: 3 15 15 
+  :header-rows: 1
+
+  * - ID
+    - Decision Information 
+    - Notes
+  * - A
+    - NEEDED  
+    -  
+  * - B
+    - If simulant is eligible, either 50% or 100% depending on scenario  
+    - For 50% scenario, assignment is random 
+  * - C
+    - NEEDED  
+    -  
+
+
+.. todo::
+  - Unclear how patients are assigned mono or combo therapy and probability of assignment 
+  - Need to figure out how treatment effect works 
+
+
+.. _uscvd4.4:
+
+4.4 Treatment Assignment and Effect Modeling
+--------------------------------------------
+
+**Treatment Assignments** 
+
+**LDL-C Treatments** 
+
+The decision to assign a patient treatment is completed in the healthcare visits above. All LDL-c treatments are 
+statins for simplicity in this model. The choice of intensity is determined by the patients ASCVD score and LDL-c. 
+
+- ASCVD is between 5 and 7.5%, patient is assigned low intensity statin 
```

Comment:
As I'm reading through this let's continue to use 'simulant' rather than 'patient' for consistency